workload/s3/mix_io/type4_mix_object_crud_operations.yaml

```diff
@@ -0,0 +1,15 @@
+test_1:
+    TEST_ID: TEST-40042
+    object_size:
+        start: 0bytes
+        end: 1Gib
+    write_percentage: 30  # Percentage of data to fill the storage.
+    read_percentage: 30  # Percentage of data to be used to read from storage.
+    delete_percentage: 15  # Percentage of data to be deleted from storage.
+    cleanup_percentage: 85  # Percentage of data once reached then cleanup all data.
+    total_storage_size: 1TiB  # Total available storage from cluster or user to be used.
```

Comment:
This should be calculated from total capacity available in hctl status.

Reply:
User capacity might be indefinite hence we have added this option for user input.

Reply:
Added code to fetch it from cluster.